src/client/store/api/api.action-creator.js

```diff
@@ -88,11 +88,14 @@ export function createApiAction(type, payloadCreator, metaCreator) {
         onResponse(dispatch, getState, response);
       }
     } catch (error) {
+      // TODO: Uncomment the bits to get the server error message into the state?
```

Comment:
The existing API action creator will result in the Axios error message being returned as `error.message`. Uncommenting these lines will allow us to use the response body, if any, that might come back from the API. Whether this is what we want to do should await a more comprehensive plan for error handling.